vars/runPipeline.groovy

```diff
@@ -445,6 +453,9 @@ def call() { timeout(time: 1, unit: 'HOURS') {
         "nomad-software/dunit",
         "repeatedly/mustache-d",
         "s-ludwig/taggedalgebraic",
+        "libmir/mir-algorithm",
+        "libmir/mir-random",
```

Comment:
What is the difference between mir-random and others that are in the switch?

Reply:
The Project Tester will try the `script` line of the `.travis.yml` by default.
That's generally preferred, because that's puts less maintenance effort here.
However, for `mir-algorithm` there would be no `travis_wait` and for `mir` we don't want to fetch `imageformats` just to test the examples (at least not while DUB is still moderately unstable).

Does this answer your question?